frontend/docs/lib/management.openapi.yaml

```diff
@@ -0,0 +1,674 @@
+openapi: 3.0.1
```

Comment:
is this copied over manually?

Reply:
Yup, unfortunately there is no easy way to cleanly generate this from our existing OpenAPI spec.